bitnami/zookeeper/values.yaml

```diff
@@ -442,6 +442,18 @@ serviceAccount:
 ## Network policies
 ## Ref: https://kubernetes.io/docs/concepts/services-networking/network-policies/
 ##
+
+sidecar: []
+  ## @param sidecars Extra containers to the pod
```

Comment:
Please move the comments before the sidecar key definition (line 447-455 to 445). This must fix the linter Readme fail that you are having.